scripts/user_details.json

```diff
@@ -1,6 +1,6 @@
 {
   "email" : {
-    "address": "Testy22@gmail.com"
+    "address": "rohannagar11@gmail.com"
```

Comment:
Change this back to `Testy22@gmail.com`?

Reply:
Done